site/data/scans/auth/output.yml

```diff
@@ -112,14 +112,6 @@ reddit:
       session: true
       verification: true
 spotify:
-  site: "https://accounts.spotify.com"
-  plans:
-    stdbba:
-      auth: true
-      username: true
-      password: true
-      session: true
-      verification: true
```

Comment:
The script in the other repo is going to need to check and skip when config.local is present I believe.

Reply:
Or the template to handle empty sites.

Reply:
https://github.com/zaproxy/zaproxy-website/pull/3005